src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -107,6 +115,19 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
   const checkUpdates = useCheckUpdates();
   const { showErrorAlert } = useContext(AlertModalContext);
   const { features, policies } = useContext(CampaignConfigContext);
+  const { selectedIdType, setSelectedIdType } = useContext(
+    IdentificationContext
+  );
+
+  const getSelectionArray = useCallback((): IdentificationFlag[] => {
+    const selectionArray = [];
+    selectionArray.push(features?.id || defaultSelectedIdType);
+    features?.alternateIds &&
+      features.alternateIds.map(alternateId =>
+        selectionArray.push(alternateId)
+      );
+    return selectionArray;
+  }, [features]);
```

Comment:
Wonder if we want to use `useMemo` in this case since the return value should likely be the same for each campaign. Noticed that on every render of this screen, `hasMultiInputSelection` is called, which triggers this.

Reply:
good catch! I have changed to `useMemo`